sdk/identity/identity/integration/AzureFunctions/RunTest/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "recommendations": [
+    "ms-azuretools.vscode-azurefunctions"
```

Comment:
I would vote to delete this, the less files the better! 

But, it's pretty localized so no strong feelings

Reply:
Oh yes, I had this from when I was trying to use VS Code for Az functions to get this working.